test/test_linalg.py

```diff
@@ -7449,6 +7449,48 @@ def test_permute_matmul(self):
         c = a.permute(0, 1, 3, 2).matmul(b)
         self.assertEqual([c.min(), c.max(), c.sum()], [24, 24, 414720])
 
+    def test_bfloat16_accumulation_with_ref_path(self):
+        # fix https://github.com/pytorch/pytorch/issues/95125
+        # and https://github.com/pytorch/pytorch/issues/83863
+        # for bf16 accumulation in gemm ref path
+        def check_correctness(fn, *args):
+                expected = fn(*args).bfloat16()
```

Comment:
For ref result, is it ok to use fp32 inputs directly or do a fake quant with `bfloat16()` and `float()` for exact match with bfloat16 result?

Reply:
We do not need fake quant here since we always have new tensors whose elements are `1`. I do not see acc loss for `[1.0,].bfloat16().float()`.